package.json

```diff
@@ -20,15 +20,18 @@
     "serve:dist": "web-dev-server --app-index dist/index.html --open",
     "start": "web-dev-server --node-resolve --open /dev",
     "start:base": "web-dev-server --node-resolve --open /dev --theme=base",
+    "start:aura": "web-dev-server --node-resolve --open /dev --theme=aura",
     "check-releases": "node ./scripts/check-releases.js && web-dev-server --node-resolve --open /scripts/check-releases.html",
     "test": "web-test-runner",
+    "test:aura": "yarn test --config web-test-runner-aura.config.js",
```

Comment:
For now this fails as there are no packages yet. I'll add a GitHub Actions workflow when we have first component with the Aura visual tests (could be done in the same PR).